cmd/submit.go

```diff
@@ -147,6 +147,32 @@ figuring things out if necessary.
 			return errors.New("no files found to submit")
 		}
 
+		// if the user submits a directory, confirm the list of files
+		if len(tx.ArgDirs) > 0 {
+			prompt := "You specified a directory. Here are the files you are submitting:\n"
+			for i, path := range paths {
+				prompt += fmt.Sprintf(" [%d]  %s\n", i+1, path)
+			}
+			prompt += "\nPress ENTER to submit, or control + c to cancel: "
+			
+			conf_q := &comms.Question{
+				Prompt: prompt, 
+				DefaultValue: "y",
+				Reader: In,
+				Writer: os.Stdout,
+			}
+			answer, err := conf_q.Ask()
+			if err != nil {
+				fmt.Println(err)
```

Comment:
This should not print the error, only return it. The error will be printed to STDERR since we're mapping the command function to `RunE`